Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -0,0 +1,5783 @@
+    """
+
+    id = args['id']
+    response = client.custom_whitelist_delete_request(id=id)
+    command_results = generate_simple_command_results('id', id, response, OutputTitles.CUSTOM_WHITELIST_DELETE)
+    return command_results
+
+
+def custom_whitelist_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """List custom whitelist members.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+    id = args.get('id')
+    response = client.custom_whitelist_list_request(id=id)
+    # formatted_response, pagination_message = paginate_results(client.version, response, args)
+    parsed_data, pagination_message, formatted_response = list_response_handler(client, response,
+                                                                                client.parser.custom_whitelist, args,
+                                                                                id)
+    readable_output = tableToMarkdown(name=OutputTitles.CUSTOM_WHITELIST_LIST,
+                                      metadata=pagination_message,
+                                      t=parsed_data,
+                                      headers=['id', 'name', 'request_url', 'path', 'domain', 'status'],
+                                      headerTransform=string_to_table_header)
+    command_results = CommandResults(readable_output=readable_output,
+                                     outputs_prefix='FortiwebVM.CustomGlobalWhitelist',
+                                     outputs_key_field='id',
+                                     outputs=parsed_data,
+                                     raw_response=response)
+    return command_results
+
+
+def custom_predifined_whitelist_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """List the Custom Predifined members.
+
+    Args:
+        client (Client): FortiwebVM API client.
+        args (Dict[str, Any]): Command arguments from XSOAR.
+
+    Returns:
+        CommandResults: outputs, readable outputs and raw response for XSOAR.
+    """
+
+    id = args.get('id')
+    object_type = args.get('type')
+    response = client.custom_predifined_whitelist_list_request()
+    if object_type:
+        data = response if client.version == ClientV1.API_VER else response['results']
+        rel_data = dict_safe_get(find_dict_in_array(data, 'type', object_type), ['details'])
+        response = rel_data if client.version == ClientV1.API_VER else {'results': rel_data}
+    else:
+        data = response if client.version == ClientV1.API_VER else response['results']
+        rel_data = [member for object_type in data for member in object_type['details']]
+        response = rel_data if client.version == ClientV1.API_VER else {'results': rel_data}
```

Comment:
The only difference here is the logic in assignment of rel_data variable. other logic can be moved out and be a common logic.